.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -7,6 +7,10 @@
 <!-- IMPORTANT ECLIPSE NOTE: If you change this file, you must restart Eclipse
  for your changes to take effect in its Checkstyle integration. -->
 <suppressions>
+    <!-- Eclipse/Idea APT generated code -->
+    <suppress files="[/\\]generated_src[/\\]" checks="." />
+    <suppress files="[/\\]src[/\\]generated[/\\]java[/\\]" checks="." />
```

Comment:
I believe the second case (src/generated) may be redundant. (And it is also redundant in Foundry)